src/ensembl/src/shared/types/thoas/product.ts

```diff
@@ -52,7 +53,7 @@ export type Product = {
   length: number;
   protein_domains: ProteinDomain[];
   external_references: ExternalReference[];
-  sequence_checksum?: string;
+  sequence?: Sequence;
```

Comment:
@azangru what about the sequence here, should it be optional or compulsory as well to keep in line with cds and cdna?